tests/examples_arguments_syntax/line_chart_with_cumsum_faceted.py

```diff
@@ -0,0 +1,36 @@
+"""
+Faceted Line Chart with Cumulative Sum
+------------------------------
```

Comment:
```suggestion
Faceted Line Chart with Cumulative Sum
--------------------------------------
```

There is a warning in the docbuild if the underline length does not match the title

Reply:
- Matched underline length with title